mkdocs.yml

```diff
@@ -243,6 +243,55 @@ plugins:
             Find a developer: Gdzie możemy znaleźć dobrego programistę lub firmę, którzy będą współpracować z OSM?
             Downloading and using:  Pobieranie i korzystanie z danych projektu OSM
 
+        - locale: es
+          name: Español
+          default: true
```

Comment:
```suggestion
```

There is only one instance of the `default: true`. This parameter is used for English, and it points to the default language of the docs.